docs/examples/audio_processing/audio_decoder.ipynb

```diff
@@ -34,7 +34,7 @@
    "source": [
     "used `batch_size` is `1`, to keep things simple.\n",
     "\n",
-    "2. Next, let's implement the pipeline. Firstly, we need to load data from disk (or any other source). FileReader is able to load data, as well as it's labels. For more information, refer to FileReader docs. Furthermore, similarly to image data, you can use Reader operators that are specific for a given dataset or a dataset format (see [CaffeReader](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops.html#nvidia.dali.ops.CaffeReader)). After loading the input data, the pipeline decodes the audio data. As stated above, the AudioDecoder operator is able to decode most of the well-known audio formats.\n",
+    "2. Next, let's implement the pipeline. Firstly, we need to load data from disk (or any other source). readers.File is able to load data, as well as it's labels. For more information, refer to readers.File docs. Furthermore, similarly to image data, you can use Reader operators that are specific for a given dataset or a dataset format (see [readers.Caffe](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/supported_ops_legacy.html#nvidia.dali.ops.readers.Caffe)). After loading the input data, the pipeline decodes the audio data. As stated above, the AudioDecoder operator is able to decode most of the well-known audio formats.\n",
```

Comment:
FYI: We are going to rewrite this with the functional API. It can stay like this for now, but most likely we are going to change it very soon.